proto/rill/runtime/v1/queries.proto

```diff
@@ -302,17 +304,41 @@ message MetricsViewAggregationMeasure {
   repeated google.protobuf.Value builtin_measure_args = 3;
 }
 
+message MetricsViewAggregationSort {
+  string name = 1;
+  bool desc = 2;
+}
+
+message MetricsViewColumnAlias {
+  string name = 1;
+  oneof alias { // Is this overkill to future proof this?
+    MetricsViewMeasureAlias measure_alias = 2;
+  }
+}
+
+message MetricsViewMeasureAlias {
+  enum MeasureType {
+    MEASURE_TYPE_UNSPECIFIED = 0;
+    MEASURE_TYPE_COUNT = 1;
+    MEASURE_TYPE_COUNT_DISTINCT = 2;
```

Comment:
These can already be defined with an alias through `MetricsViewAggregationMeasure`